support/setup_local_env.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# set -euo pipefail
+
+export HAB_LICENSE="accept-no-persist"
+sudo -E hab pkg install core/bzip2
+sudo -E hab pkg install core/libarchive
+sudo -E hab pkg install core/libsodium
+sudo -E hab pkg install core/openssl
+sudo -E hab pkg install core/xz
+sudo -E hab pkg install core/zlib
+sudo -E hab pkg install core/zeromq
+sudo -E hab pkg install core/protobuf --binlink
+sudo -E hab pkg install core/rust --binlink
```

Comment:
As a Rust developer, I don't think I would ever want to binlink `core/rust` on my local development system, since that would be a global change, and `rustup` provides a ton of flexibility for developers.

If someone doing other Rust development were to run this, it could cause headaches for their other projects.